docs/platforms-release-process.md

```diff
@@ -86,30 +157,46 @@ Ensure all dependencies and subdependencies have Apache-compatible licenses.
     coho check-license -r android
 
 ## Prepare Release
-Increase the version within package.json using SemVer, and remove the `-dev` suffix.
+
+### Remove the `-dev` suffix from version
+
+This command removes `-dev` from the `version` entry in `package.json`:
 
     for l in cordova-android; do ( cd $l; v="$(grep '"version"' package.json | cut -d'"' -f4)"; if [[ $v = *-dev ]]; then v2="${v%-dev}"; echo "$l: Setting version to $v2"; sed -i '' -E 's/version":.*/version": "'$v2'",/' package.json; fi) ; done
 
-In `cordova-android`, also remember to bump the version in `framework/build.gradle`.
+Note: This command [doesn't actually work](https://issues.apache.org/jira/browse/CB-13809). You can also replace `-dev` manually of course.
```

Comment:
Strange, it worked fine for me on this latest android release.

Reply:
Yeah, I also felt this was strange as I assumes these were used quite often. You can take a look at the linked issues.